server/enclave/enc.cpp

```diff
@@ -0,0 +1,99 @@
+// Include the trusted modelaggregator header that is generated
```

Comment:
What does the name of this file, `enc.cpp` stand for? Maybe we can change it to something like `ecalls.cpp`?